public/api.php

```diff
@@ -10,18 +10,10 @@
  * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  */
 
-/** @deprecated Use MAHO_ROOT_DIR instead. */
-define('MAGENTO_ROOT', dirname(__DIR__));
 define('MAHO_ROOT_DIR', dirname(__DIR__));
 define('MAHO_PUBLIC_DIR', __DIR__);
 
-if (file_exists(MAHO_ROOT_DIR . '/app/bootstrap.php')) {
-    require MAHO_ROOT_DIR . '/app/bootstrap.php';
-    require MAHO_ROOT_DIR . '/app/Mage.php';
-} else {
-    require MAHO_ROOT_DIR . '/vendor/mahocommerce/maho/app/bootstrap.php';
-    require MAHO_ROOT_DIR . '/vendor/mahocommerce/maho/app/Mage.php';
-}
+require MAHO_ROOT_DIR . '/vendor/autoload.php';
```

Comment:
loving this semplification

Reply:
Absolutely, it's nice that everything is auto-loadable now.

Technically there's now some similar code in `app/bootstrap.php` to find `vendor/autoload.php`, but it will never be executed in a vanilla project. It's only there for backwards compatibility with custom scripts that don't use composer.